app/models/article.rb

```diff
@@ -1,2 +1,17 @@
 class Article < ApplicationRecord
+  def even?
+    Time.current.to_i.even?
+  end
+
+  def odd?
+    Time.current.to_i.odd?
+  end
+
+  def odd_or_even?
+    Time.current.to_i.odd? || Time.current.to_i.even?
+  end
+
+  def not_odd_or_even?
+    !(Time.current.to_i.odd? || Time.current.to_i.even?)
+  end
```

Comment:
⚠️ **[undercover]** <Untested instance method> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Instance method `not_odd_or_even?` is missing coverage for line 15 (node coverage: 0.0)
<!-- __reviewdog__:ChBhYWNiMWZlODE3NzI5NDQ0Egp1bmRlcmNvdmVy -->